addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -273,12 +278,32 @@ private synchronized void refreshData() {
 	public void onConnectionLost() {
 		logger.info("Bridge connection lost. Updating thing status to OFFLINE.");
 		previousOnline = false;
-		updateStatus(ThingStatus.OFFLINE);
+		updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR);
+		for (Thing thing : getThing().getThings()) {
+			ThingHandler handler = thing.getHandler();
+			if (handler != null && handler instanceof MaxDevicesHandler ) {
+				((MaxDevicesHandler) handler).setOffLine();
```

Comment:
I would have thought that the status of child things is automatically set by the framework to OFFLINE, if a bridge goes OFFLINE. Is this not the case?